mezlet/iReport-React
# #164030541 set up webpack

**What does this PR do?**
Sets up webpack and react. Configures webpack forSASS loading 

**Description of Task to be completed?**
-setup webpack
-setup webpack devserver
-configure webpack scss loading
-setup react
-set up express server
-configure linter


**How should this be manually tested?**
 - clone repo locally
 - checkout `ch-setup-webpack`
 - define a `PORT` variable in `.env` file in the project folder
 - run `npm run devstart`

The bare-bones react app should be launched

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
#164030541 

**Screenshots (if appropriate)**
